src/controllers/tasks/registered.office.address.controller.ts

```diff
@@ -30,8 +29,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
 
 export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
+    const companyNumber = urlUtils.getCompanyNumberFromRequestParams(req);
```

Comment:
you can move lines 32,33 to just before 50, it will save us an api call radio button is yes or no

Reply:
amended it now